flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTest.java

```diff
@@ -1420,6 +1422,13 @@ public void testRequestPartitionState() throws Exception {
         }
     }
 
+    private static ExecutionAttemptID deepCopyExecutionAttemptId(ExecutionAttemptID toCopy)
+            throws IOException, ClassNotFoundException {
+        return InstantiationUtil.deserializeObject(
+                InstantiationUtil.serializeObject(toCopy),
+                ExecutionGraphTestUtils.class.getClassLoader());
+    }
+
```

Comment:
Why aren't you directly using `InstantiationUtils#clone`?

Reply:
Good idea! I didn't realized there is a `InstantiationUtils#clone`. Will remove this method and directly use it.